internal/subshell/subshell_win.go

```diff
@@ -3,8 +3,30 @@
 
 package subshell
 
-import "github.com/ActiveState/cli/internal/subshell/cmd"
+import (
+	"fmt"
+
+	"github.com/ActiveState/cli/internal/subshell/bash"
+	"github.com/ActiveState/cli/internal/subshell/cmd"
+	"github.com/ActiveState/cli/internal/subshell/pwsh"
+)
 
 var supportedShells = []SubShell{
 	&cmd.SubShell{},
+	&pwsh.SubShell{},
+	&bash.SubShell{},
+}
+
+const (
+	SHELL_ENV_VAR = "COMSPEC"
+	OS_DEFULAT    = "cmd.exe"
+)
+
+func supportedShellName(name string) bool {
+	for _, subshell := range supportedShells {
+		if name == fmt.Sprintf("%s.exe", subshell.Shell()) {
```

Comment:
Suggest renaming `name` to `filename` to make it explicit that this is dealing in shell filenames and not shell names. Also please make the assertion case insensitive (I seem to recall windows sometimes refers to it as `CMD.EXE`, and windows is case-insensitive).